synapse/rest/client/capabilities.py

```diff
@@ -62,21 +72,10 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
                 "org.matrix.msc3244.room_capabilities"
             ] = MSC3244_CAPABILITIES
 
-        if self.config.experimental.msc3283_enabled:
-            response["capabilities"]["org.matrix.msc3283.set_displayname"] = {
-                "enabled": self.config.registration.enable_set_displayname
-            }
-            response["capabilities"]["org.matrix.msc3283.set_avatar_url"] = {
-                "enabled": self.config.registration.enable_set_avatar_url
-            }
-            response["capabilities"]["org.matrix.msc3283.3pid_changes"] = {
-                "enabled": self.config.registration.enable_3pid_changes
-            }
-
```

Comment:
Do we want to keep adding these for a version or two to give a small window of backwards compat? Not sure if we care enough to bother?

If we do keep it we should add an issue and add it to the next month milestone for us to remember to rip this out

Reply:
> Do we want to keep adding these for a version or two to give a small window of backwards compat? Not sure if we care enough to bother?

I have no preferences here. Is there a standard procedure for Synapse? This kind of change is surely not the first, is it?

Reply:
Have asked in #synapse-dev:matrix.org, will see if anyone has opinions.

Reply:
The conclusion we came to is that we should leave this in for one release, and then rip it out. If you could make the change that would be grand, and I can file an issue and put in the right place in our tracker to make sure it doesn't get forgotten :slightly_smiling_face: